src/Clapi/Lib.hs

```diff
@@ -1,9 +0,0 @@
-{-# LANGUAGE DeriveDataTypeable #-}
```

Comment:
Don't you need to remove this file from the `cabal` file too?

Reply:
Nope, because it wasn't there.

Reply:
Ah, how strange!